### PR TITLE
vagrant service-manager 1.3.0 release

### PR DESCRIPTION
**Pull request by budhrg**

### CODE REVIEW

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: vagrant-service-manager 1.3.0 released focusing on Kubernetes support
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: vagrant-service-manager 1.3.0 Released: Focus on Kubernetes Support

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+author: budhram
+date: 2016-08-16 12:54:00 UTC
+tags: vagrant, devtools, releases, kubernetes
+published: true
+comments: true
+---
+
+This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes setup information which enable users to access Kubernetes server running inside ADB VM from host.
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes configuration information.  This enable users to access the Kubernetes server running inside ADB virtual machine from their host.

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+We also extended the client binary installation support for 'Kubernetes' in this release. Not only that, users now can install these client binaries in [CDK (Container Development Kit)](http://developers.redhat.com/products/cdk/overview/) platform too. The overview of the client binary installation feature can be found in the previous blog ['Client Binary Installation Now Included in the ADB'](blog/2016/07/vagrant-service-manager-install-cli).
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: Client binary installation support for 'Kubernetes' is included in this release. Support for client binary installs has also been extended to users of the [Red Hat Container Development Kit](http://developers.redhat.com/products/cdk/overview/). Information about client binary installation can be found in the previous release announcement ["Client Binary Installation Now Included in the ADB"](blog/2016/07/vagrant-service-manager-install-cli).

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+The main features of this release are:
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: The main new features of this release are:

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+* Enable displaying of Kubernetes setup information so one can access Kubernetes running inside ADB VM
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Configuration information for Kubernetes provided as part of the `env` command

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+* Client binary installation support for Docker, OpenShift CDK
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Client binary installation support for OpenShift, Kubernetes, and docker in the Red Hat Container Development Kit

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+* Client binary installation support for Kubernetes in both ADB/CDK
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Client binary installation support for Kubernetes added to the ADB

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+* Auto detection of already downloaded executable `oc.exe` binary in Windows
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Auto-detection of a previously downloaded `oc` executable binary on Windows

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+* Added unit and acceptance tests for Kubernetes service
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Unit and acceptance tests for the Kubernetes service

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+* Enabled 'kubernetes' service from Vagrantfile configuration option as
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Option to enable `kubernetes` from a Vagrantfile as follows:

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+```
+  config.servicemanager.services = 'kubernetes'
+```
+
+## `Install Kubernetes client binary`
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: ## Install Kubernetes Client Binary

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: The command below will install the `kubernetes` binary, `kubctl`:

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+```
+
+**Configure your shell as:**
+
+```
+$ eval "$(VAGRANT_NO_COLOR=1 vagrant service-manager install-cli kubernetes | tr -d '\r')"
+```
+
+## `Enable access to Kubernetes server running inside ADB VM`
```
Comment (bexelbie) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: ## Enable Access to the Kubernetes Server Running Inside of the ADB

**Comment by bexelbie** [comment]
I've attached some suggestions/edits.

**Comment by budhrg** [comment]
@bexelbie Updated PR as per your suggestions.

**Comment by jberkus** [comment]
Merged after some edits.

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
+---
+title: vagrant-service-manager 1.3.0 Released - Focus on Kubernetes Support
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: This title is a little too long and should state the official name and not the package name, something like:

"Vagrant Service Manager 1.3.0 Released"

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+author: budhram
+date: 2016-08-16 12:54:00 UTC
+tags: vagrant, devtools, releases, kubernetes
+published: true
+comments: true
+---
+
+This release of [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) introduces support for displaying Kubernetes configuration information. This enable users to access the Kubernetes server running inside ADB virtual machine from their host.
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * release > version
* running > that runs
* "their" is ambiguous, not clear whether it refers to the server host or the virtual machine host

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+Client binary installation support for 'Kubernetes' is included in this release. Support for client binary installation has also been extended to users of the [Red Hat Container Development Kit](http://developers.redhat.com/products/cdk/overview). Information about client binary installation can be found in the previous release announcement ["Client Binary Installation Now Included in the ADB"](../../../../blog/2016/07/vagrant-service-manager-install-cli).
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * release > version
* The first sentence can be modified to match the first paragraph, since it's a continuation of the release "highlights", so something like "This version also includes binary installation support for Kubernetes." (no need to use quotes for Kubernetes)
* The second sentence can be modified for consistency, something like "This support is extended to users of...."
* "For information about .... , see... " (style consistency)

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+
+The main new features of this release are:
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: If we already list 2 release highlights above, we can change this sentence to say "full list of features". Also, if we're enhancing existing features and not just adding new features, then the word "new" is not necessary.

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+
+* Configuration information for Kubernetes provided as part of the `env` command
+* Client binary installation support for OpenShift, Kubernetes and docker in the Red Hat Container Development Kit
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Is "OpenShift" applicable for both Enterprise and Online?
* docker > Docker

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+* Client binary installation support for Kubernetes added to the ADB
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: maybe move this bullet point to be 2nd, so that it matches that order of the release highlights above the list

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+* Auto-detection of a previously downloaded `oc` executable binary on Windows
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * Windows operating systems (should be more than one?)

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 21]
```diff
@@ -0,0 +1,67 @@
...
+* Unit and acceptance tests for the Kubernetes service
+* Option to enable kubernetes from a Vagrantfile as follows:
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * as follows > with the following command
* Is it appropriate to list a command in this list? do we document this anywhere? it actually doesn't look like a command, I'm not familiar with the context but this looks more like a parameter/variable

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 22]
```diff
@@ -0,0 +1,67 @@
...
+
+```
+  config.servicemanager.services = 'kubernetes'
+```
+
+## Install Kubernetes Client Binary
+
+The command below will install the kubernetes binary, `kubectl`:
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * we try not to use above/below, maybe rephrase to something like "Run the following command to install the kubectl binary"
* kubernetes > Kubernetes

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 23]
```diff
@@ -0,0 +1,67 @@
...
+# run binary as:
+# kubectl <command>
+export PATH=/home/budhram/.vagrant.d/data/service-manager/bin/kubernetes/1.2.0:$PATH
+
+# run following command to configure your shell:
+# eval "$(VAGRANT_NO_COLOR=1 vagrant service-manager install-cli kubernetes | tr -d '\r')"
+
+```
+
+**Configure your shell as:**
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * remove bold tagging and rephrase as a task step, something like "Run the following command to configure your shell"
* "as" seems like an incomplete statement (as what?)

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 24]
```diff
@@ -0,0 +1,67 @@
...
+* Client binary installation support for Kubernetes added to the ADB
+* Auto-detection of a previously downloaded `oc` executable binary on Windows
+* Unit and acceptance tests for the Kubernetes service
+* Option to enable kubernetes from a Vagrantfile as follows:
+
+```
+  config.servicemanager.services = 'kubernetes'
+```
+
+## Install Kubernetes Client Binary
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: consider numbering the task steps if they need to be followed in order

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 25]
```diff
@@ -0,0 +1,67 @@
...
+
+```
+
+**Configure your shell as:**
+
+```
+$ eval "$(VAGRANT_NO_COLOR=1 vagrant service-manager install-cli kubernetes | tr -d '\r')"
+```
+
+## Enable Access to the Kubernetes Server Running Inside of the ADB
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * change to sentence-caps (leave capitalization on Kubernetes and ADB)

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 26]
```diff
@@ -0,0 +1,67 @@
...
+
+```
+$ vagrant service-manager env kubernetes
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: needs lead-in sentence similar to the previous task, something like "Run the following command to.."

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 27]
```diff
@@ -0,0 +1,67 @@
...
+# Set the following environment variables to enable access to the
+# kubernetes server running inside of the vagrant virtual machine:
+export KUBECONFIG=/home/budhram/.vagrant.d/data/service-manager/kubeconfig
+
+# run following command to configure your shell:
+# eval "$(vagrant service-manager env kubernetes)"
+```
+
+**Configure shell as:**
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: same comment as previous

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md [hunk 28]
```diff
@@ -0,0 +1,67 @@
...
+
+```
+eval "$(vagrant service-manager env kubernetes)"
+```
+
+For a full list of changes in version 1.3.0 please see [the release log](https://github.com/projectatomic/vagrant-service-manager/releases/tag/v1.3.0).
```
Comment (thatdocslady) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: * remove "please"
* add a comma (,) before "see"

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md (hunk 18 above)
Comment (budhrg) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: @thatdocslady: Only Enterprise

File: source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md (hunk 25 above)
Comment (budhrg) [inline source/blog/2016-08-16-vagrant-service-manager-1-3-0-release.html.md]: @thatdocslady: Can't remove capitalization for ADB as it is abbreviation for Atomic Developer Bundle.

Others are updated as suggested :+1:

**Comment by budhrg** [comment]
@thatdocslady: Sent [new PR](https://github.com/projectatomic/atomic-site/pull/352) with the changes you suggested.
